extension/page-detect.js

```diff
@@ -11,10 +11,14 @@ window.pageDetect = (() => {
 
 	const isRepoTree = () => isRepo() && /\/tree\//.test(getRepoPath());
 
+	const isIssueSearch = () => location.pathname.indexOf('/issues') === 0;
```

Comment:
@busches Indeed, but ESLint doesn't know that this is a string, it could be an Array too... JS is cool, but a good type system would have been wonderful.

https://github.com/sindresorhus/eslint-plugin-unicorn/pull/64#issuecomment-269987212